generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-cxf-soap-grouped/pom.xml

```diff
@@ -73,7 +73,7 @@
     <dependency>
       <groupId>io.quarkiverse.cxf</groupId>
       <artifactId>quarkus-cxf-test-util</artifactId>
-      <version>${quarkus-cxf.version}</version>
+      <version>3.8.3</version>
```

Comment:
@gsmet pointed me to this. Per se, it is not correct but not that important, because it is a test dependency in a test project. The main thing is, that the BOMs are correct. Hard to say, whether it is a bug in the generator plugin. In this situation, where quarkus-cxf-test-util is not managed and the version of all qcxf artifacts in `org.apache.camel.quarkus:camel-quarkus-bom` is 3.8.3, the generator can't perhaps do any other guess. WDYT, @aloubyansky? The bottom line is that both QCXF and CQ have their own test BOMs which we ideally might want to align and use in the Platform, but it is perhaps not worth of the effort, to solve such small details as this one.

Reply:
If a test dependency isn't managed by platform BOMs then its version is copied from the original test POM for that test.

Reply:
I see. Then it works as designed. What is your take on aligning and using the test BOMs? It is not worth the trouble, is it?

Reply:
```
test BOMs which we ideally might want to align and use in the Platform
```
could you please clarify what you mean specifically?

Reply:
Here are the two test boms: 
https://github.com/quarkiverse/quarkus-cxf/blob/main/bom-test/pom.xml#L21-L25
https://github.com/apache/camel-quarkus/blob/main/poms/bom-test/pom.xml#L65-L71

The CQ test BOM imports the QCXF test BOM so to avoid this kind of issues, we'd need to not only use the BOMs in the generated tests in the Platform, but we also would have to align the two BOMs so that newer QCXF gets propagated to the CQ test BOM. As I said this is perhaps too much work for too little benefit.